src/BlazorStrap/Components/ListGroup/BSListGroupItem.razor.cs

```diff
@@ -14,6 +14,7 @@ public abstract class BSListGroupItemBase : ComponentBase
             .AddClass($"list-group-item-{Color.ToDescriptionString()}", Color != Color.None)
             .AddClass("active", IsActive)
             .AddClass("disabled", IsDisabled && ListGroupType != ListGroupType.Button)
+            .AddClass("list-group-item-action", ListGroupType != ListGroupType.List)
```

Comment:
This class appears on a or button tag so should be.
.AddClass("list-group-item-action", ListGroupType != ListGroupType.List && ( ListGroupType.Button || ListGroupType.Link) ) 
Will approve with requested changed.

Reply:
I understand your point but we only have 3 ListGroupType

![image](https://user-images.githubusercontent.com/5600102/79392164-ec75d080-7f40-11ea-9519-06fc51eb9778.png)

1. Button - Actionable
2. Link - Actionable
3. List - Non Actionable

So adding the conditions suggested won't make any difference unless a new ListGroupType item is added, cause the only non actionable is the ListGroupType.List.

I will add it anyway and it should go like this:

```
.AddClass("list-group-item-action", ListGroupType != ListGroupType.List && 
                                                (ListGroupType == ListGroupType.Button || 
                                                ListGroupType == ListGroupType.Link))
```